helpers/loyverse.py

```diff
@@ -1,133 +1,109 @@
 import requests
 import json
+from typing import Dict, Optional
 
+from helpers.points import Points
 
-class LoyverseConnector:
-    BASE_URL = "https://api.loyverse.com/v1.0"
-    READ_ALL_CUSTOMERS_ENDPOINT = f"{BASE_URL}/customers?updated_at_min=2023-07-01T12:30:00.000Z&limit=250"
-    CREATE_OR_UPDATE_CUSTOMER_ENDPOINT = f"{BASE_URL}/customers"
-
-    def __init__(self, token):
-        self.token = token
-
-    @staticmethod
-    def is_number(variable):
-        return isinstance(variable, (int, float))
 
-    @staticmethod
-    def get_customer_from_username(customers, username):
-        customer = customers.get(username)
-        if customer:
-            return customer
-        else:
-            raise Exception(f"@{username} is not a recognised user, try again!")
+def _default(obj):
+    if hasattr(obj, 'to_json'):
+        return obj.to_json()
+    raise TypeError(f'Object of type {obj.__class__.__name__} is not JSON serializable')
 
-    @staticmethod
-    def read_customers(data):
-        customers = {}
 
-        for c in data.get("customers"):
-            note = c.get("note")
-            if note:
-                infos = {
-                    "note": note,
-                    "id": c.get("id"),
-                    "name": c.get("name"),
-                    "total_points": c.get("total_points")
-                }
+class InsufficientFundsError(Exception):
+    pass
 
-                customers[note] = infos
 
-        return customers
+class Customer:
```

Comment:
we should implement the logic for Access checker in this class

Reply:
I tried, but it's a bit trickier than it seems. This is something that I will address in a future pull request, one which includes database functionality.